### PR TITLE
Update scenario.md

### PR DESCRIPTION
**Pull request by RymMichaut**
Try to make the description a bit clearer and simpler

### CODE REVIEW

**Comment by FlorianJacta** [comment]
What if we put the use cases before explaining a scenario? Having these use cases makes us understand what they are directly

File: docs/manuals/core/concepts/scenario.md [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 
-As its name implies, with Taipy scenarios, the users can instantiate different versions of a business
-problem with different assumptions. This is extremely useful in a business context where impact analysis
-and what-if analysis are essential in the decision process.
+A Taipy scenario represents a business problem with consistent data and parameters.
 
-After analyzing its first scenario, an end-user may be interested in modifying input
-data nodes (not the intermediate nor the output data nodes), re-running the same pipelines, and
-comparing the results with the previous run.
+It serves as a powerful tool to create different versions of a problem by incorporating assumptions. This flexibility is particularly valuable for impact analysis and decision-making processes, enabling users to create, store, edit, and execute multiple scenarios within the same application.
```
Comment (jrobinAV) [inline docs/manuals/core/concepts/scenario.md]: I don't like this line. What do you think about the following

```suggestion
Scenarios are a powerful tool to create different versions of a problem under different assumptions. This is particularly valuable for what-if analysis in decision-making processes, enabling users to create, store, edit, and execute multiple scenarios with various parameters within the same application.
```

File: docs/manuals/core/concepts/scenario.md [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
 
-For this purpose, he needs to instantiate a second scenario, execute it and compare it with the
-first scenario. This process can be repeated across multiple scenarios.
+Once an initial scenario has been analyzed, users can modify the input data nodes (excluding the intermediates and output data nodes), rerun his pipelines, and compare results.
```
Comment (jrobinAV) [inline docs/manuals/core/concepts/scenario.md]:
```suggestion
Once an initial scenario has been created and analyzed, users may want to rerun their pipelines with different input data and compare results.
```

File: docs/manuals/core/concepts/scenario.md [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
 
+This involves instantiating a second scenario, executing it, and comparing the outcomes with the first scenario.
```
Comment (jrobinAV) [inline docs/manuals/core/concepts/scenario.md]:
```suggestion
This involves instantiating a second scenario, changing the input data, executing it, and comparing the outcomes with the first scenario.
```

**Review by jrobinAV — COMMENTED** [review]
I believe it s much better. I have a few suggestions though. What do you think?

File: docs/manuals/core/concepts/scenario.md [hunk 4]
```diff
@@ -1,17 +1,14 @@
-We introduce here the important concept of a `Scenario^`. A Taipy Scenario represents an instance of a
-business problem to solve on consistent data and parameter sets.
+Introducing the `Scenario`, a fundamental key concept in Taipy.
 
-As its name implies, with Taipy scenarios, the users can instantiate different versions of a business
-problem with different assumptions. This is extremely useful in a business context where impact analysis
-and what-if analysis are essential in the decision process.
+A Taipy scenario represents a business problem with consistent data and parameters.
```
Comment (FabienLelaquais) [inline docs/manuals/core/concepts/scenario.md]: I would remove 'Taipy' here. Maybe use italics for the first occurrence of the term 'scenario'.

File: docs/manuals/core/concepts/scenario.md (hunk 1 above)
Comment (FabienLelaquais) [inline docs/manuals/core/concepts/scenario.md]: 'particularly' -> 'especially'.
Although correct, I personally seldom see that.

File: docs/manuals/core/concepts/scenario.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/core/concepts/scenario.md]: Introducing 'pipeline' here may be confusing: scenarios at this point have not been describes as holding pipelines...

File: docs/manuals/core/concepts/scenario.md [hunk 5]
```diff
@@ -20,9 +17,13 @@ first scenario. This process can be repeated across multiple scenarios.
...
     ![scenarios](../pic/scenarios.svg){ align=left }
 
-A scenario represents one instance of a business problem to solve. Each new business problem instance
-is represented by a new scenario. With Taipy, end-users can create, store, edit, and
-execute various scenarios within the same application.
+
+Two use-cases arise from the utilization of Taipy scenarios:
+
+### - Use case 1 : 
+Each _scenario_ represents a distinct instance of a business problem.
```
Comment (FabienLelaquais) [inline docs/manuals/core/concepts/scenario.md]: The "new" convention is to use stars (\*) to mark italics.
That's for a clearer support of Python symbols that can contain underscores, but not stars.

File: docs/manuals/core/concepts/scenario.md [hunk 6]
```diff
@@ -39,20 +40,16 @@ execute various scenarios within the same application.
     Then the end-user creates another scenario for February using the new information provided for the
     February period. And so on.
 
-Two _scenarios_ can also represent the same instance of a business problem but with two different sets of
-assumptions.
+### - Use case 2:
+Two _scenarios_ represent the same business problem instance, but with different sets of assumptions.
```
Comment (FabienLelaquais) [inline docs/manuals/core/concepts/scenario.md]: Same here : \*scenarios\*

File: docs/manuals/core/concepts/scenario.md [hunk 7]
```diff
@@ -39,20 +40,16 @@ execute various scenarios within the same application.
...
 
 !!! example
 
-    The end-user would like to simulate the impact of our capacity data on production planning for the
-    February use case.
+    The end-user wants to simulate how our capacity data affects production planning for the February situation.
```
Comment (FabienLelaquais) [inline docs/manuals/core/concepts/scenario.md]: "how our capacity" -> "how the capacity"?

**Review by FabienLelaquais — APPROVED** [review]
Minor changes.

Make sure you don't have lines longer than 100 characters: makes editing far easier. Thx

**Pull request merged**